### PR TITLE
Get started UI

### PR DESCRIPTION
**Pull request by SvetaMemphis**
Fix next disable when back to step 1

### CODE REVIEW

File: ui_src/src/domain/overview/getStarted/createAppUser/index.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const CreateAppUser = (props) => {
     useEffect(() => {
         createStationFormRef.current = onNext;
         getStartedDispatch({ type: 'SET_NEXT_DISABLE', payload: true });
-        getStartedDispatch({ type: 'SET_CREATE_APP_USER_DISABLE', payload: false });
+        // getStartedDispatch({ type: 'SET_CREATE_APP_USER_DISABLE', payload: false });
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/getStarted/createAppUser/index.js:54]: remove comments

File: ui_src/src/domain/overview/getStarted/createStation/index.js [hunk 2]
```diff
@@ -29,20 +29,27 @@ const CreateStation = ({ createStationFormRef }) => {
     const [getStartedState, getStartedDispatch] = useContext(GetStartedStoreContext);
 
     useEffect(() => {
-        getStartedState?.completedSteps === 0 && getStartedDispatch({ type: 'SET_NEXT_DISABLE', payload: true });
+        getStartedState?.completedSteps === 0
+            ? getStartedDispatch({ type: 'SET_NEXT_DISABLE', payload: true })
+            : getStartedDispatch({ type: 'SET_NEXT_DISABLE', payload: false });
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/getStarted/createStation/index.js:34]: getStartedDispatch({ type: 'SET_NEXT_DISABLE', payload: getStartedState?.completedSteps === 0 || false });

File: ui_src/src/domain/overview/getStarted/createStation/index.js [hunk 3]
```diff
@@ -29,20 +29,27 @@ const CreateStation = ({ createStationFormRef }) => {
...
+            getStartedDispatch({ type: 'SET_NEXT_DISABLE', payload: false });
+        }
     };
 
     const updateFormState = (field, value) => {
-        if (field === 'name')
+        if (field === 'name') {
             value.length > 0 ? getStartedDispatch({ type: 'SET_NEXT_DISABLE', payload: false }) : getStartedDispatch({ type: 'SET_NEXT_DISABLE', payload: true });
-
+        }
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/getStarted/createStation/index.js:52]: getStartedDispatch({ type: 'SET_NEXT_DISABLE', payload: value.length === 0 || false }) :

File: ui_src/src/domain/overview/getStarted/createStation/index.js (hunk 2 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/overview/getStarted/createStation/index.js:34]: Done

File: ui_src/src/domain/overview/getStarted/createStation/index.js (hunk 3 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/overview/getStarted/createStation/index.js:52]: Done

**Pull request merged**